lessons/pydata/scikitlearn_api/index.ipynb

```diff
@@ -0,0 +1,1207 @@
+{
```

Comment:
<p>Tady bych to možná trošku více rozvedl. Proč je něco takového potřeba a jaké máme možnosti a proč potřebujeme samostatný scaler pro predikované hodnoty.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>